shared/components/ResourceDetails/ResourceDetails.js

```diff
@@ -134,8 +134,11 @@ function Resource({
   silentRefetch,
   updateResourceMutation,
   windowTitle,
+  resourceTitle,
```

Comment:
I don't know what was the agreement, so correct me if I'm wrong. 
Shouldn't we display in the Busola exactly what's on a cluster? So if a resource is called `Subscription` then we should show `Subscription`, not `Event Subscription`? In theory, it's a small difference, I just wonder if it could confuse the user that those are different resources.